apache-pulsar/README.md

```diff
@@ -0,0 +1,17 @@
+# Streaming Postgres Database Changes to Apache pulsar Using Debezium
```

Comment:
Could you expand the README a bit, so it also discusses how to start a Pulsar broker (e.g. via Docker) and how to consume messages sent from Debezium, e.g. using something similar to Kafka Connect's console consumer?